lib/CodeGen/ItaniumCXXABI.cpp

```diff
@@ -2828,6 +2828,9 @@ void ItaniumRTTIBuilder::BuildVTablePointer(const Type *Ty) {
   case Type::Pipe:
     llvm_unreachable("Pipe types shouldn't get here");
 
+  case Type::TypeVariable:
+    llvm_unreachable("Type Variable shouldn't get here");
```

Comment:
I think it should be a "not yet implemented message", not "type variable shouldn't get here.".